editor/project_manager/project_dialog.h

```diff
@@ -65,67 +66,76 @@ class ProjectDialog : public ConfirmationDialog {
 	Mode mode = MODE_NEW;
 	bool is_folder_empty = true;
 
-	Button *browse = nullptr;
+	CheckButton *create_dir = nullptr;
+	Button *project_browse = nullptr;
 	Button *install_browse = nullptr;
-	Button *create_dir = nullptr;
-	VBoxContainer *name_container = nullptr;
-	VBoxContainer *path_container = nullptr;
-	VBoxContainer *install_path_container = nullptr;
+	Container *name_container = nullptr;
+	Container *project_path_container = nullptr;
+	Container *install_path_container = nullptr;
 
-	VBoxContainer *renderer_container = nullptr;
+	Container *renderer_container = nullptr;
```

Comment:
Why were those changes to non-specific container types? They still seem to have only either `HBoxContainer` or `VBoxContainer` assigned.